projects/igniteui-angular/src/lib/select/ISelectComponent.ts

```diff
@@ -0,0 +1,6 @@
+import { IgxSelectItemComponent } from './select-item.component';
+import { QueryList } from '@angular/core';
+
+export interface ISelectComponent {
```

Comment:
@IvayloG Also: write up the props/methods used in navigation directive and use this interface there instead of `public target: IgxSelectComponent;` (as that causes circular ref from the directive to the component that also imports the directive)